tests/test_compress_paths.py

```diff
@@ -2,11 +2,14 @@
 # Boundary Conditions: Tests that challenge the edges of input domains and operational boundaries.
 # Error Conditions: Tests that ensure the function handles errors gracefully, such as bad input values.
 # Performance: If necessary, tests that evaluate the function’s performance to ensure it meets required standards.
+from connectome_interpreter.compress_paths import compress_paths, compress_paths_memeff
 import unittest
 import numpy as np
 from scipy.sparse import csr_matrix
 # Adjust the import according to your package structure
-from connectome_interpreter.compress_paths import compress_paths
+import sys
+# setting path
+sys.path.append('../connectome_interpreter')
```

Comment:
This only works if your working directory, when you invoke `pytest`, is `tests`. 
```suggestion
from pathlib import Path  # but, alphabetise your imports
sys.path.append(Path(__file__).parent.parent / 'connectome_interpreter')
```

I'm guessing you need this because you haven't `pip install connectome_interpreter`'d, or you want to force Python to use the local version rather than an installed version?

Reply:
I think this came from @MonEstCha.

Reply:
Oh, my bad. I somehow couldn't get it running a different way on my local machine. Please ignore this change.